scripts/benchmarks_results/results.csv

```diff
@@ -0,0 +1,2 @@
+commit_sha,total_nodes,da_committee_size,transactions_per_round,transaction_size,rounds,leader_election_type,avg_latency_in_sec,minimum_latency_in_sec,maximum_latency_in_sec,throughput_bytes_per_sec,total_transactions_committed,total_time_elapsed_in_sec,total_num_views,failed_num_views
```

Comment:
Is this intended to be checked in?

Reply:
Yes! If you want to do test, you can run the command `just async_std example all-webserver -- --config_file ./crates/orchestrator/run-config.toml --commit_sha [ANY STRING]` and then check this file to see whether it reflects the right config setting.